modules/beta-autopilot-private-cluster/main.tf

```diff
@@ -151,12 +122,12 @@ locals {
   }]
   # BETA features
   cluster_istio_enabled                    = !local.cluster_output_istio_disabled
-  cluster_cloudrun_enabled                 = var.cloudrun
   cluster_dns_cache_enabled                = var.dns_cache
   cluster_telemetry_type_is_set            = var.cluster_telemetry_type != null
   cluster_pod_security_policy_enabled      = local.cluster_output_pod_security_policy_enabled
   cluster_intranode_visibility_enabled     = local.cluster_output_intranode_visbility_enabled
   cluster_vertical_pod_autoscaling_enabled = local.cluster_output_vertical_pod_autoscaling_enabled
+  cluster_gce_pd_csi_config                = var.gce_pd_csi_driver ? [{ enabled = true }] : [{ enabled = false }]
```

Comment:
This is always enabled in autopilot: https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/gce-pd-csi-driver

We should either not make it a variable at all, or make it a variable which only accepts `true`.

Reply:
Good catch! I did not make it a variable since users wont need to change this.